dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/persistence/file/advanced/FeatureFileOrchestrator.kt

```diff
@@ -64,7 +64,7 @@ internal class FeatureFileOrchestrator(
     )
 
     companion object {
-        private const val BASE_DIR_NAME_REG_EX = "([a-z]+[-|_])+"
+        private const val BASE_DIR_NAME_REG_EX = "([a-z]+-)+"
```

Comment:
I'm not sure if it is simpler, but probably it just can be `[a-z\-]+`. Yes, it allows to have multiple `-` or even only `-`, but maybe evaluation will be simpler/faster. I checked `([a-z]+-)+v[0-9]+` and for test string `session-replay-v2` it captures 2 groups (well, not a bit deal, since overall string still matches).